src/components/portrait/portrait-gravatar.component.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MD5 from 'crypto-js/md5';
+
+class PortraitGravatar extends React.Component {
+  static propTypes = {
```

Comment:
Can we move it the bottom of the component? And extract it from static?
```
PortraitGravatar.propTypes = {}
```

Reply:
I could certainly do that. Doing a quick grep of the `src/` directory, it looks like there are:

* ~72 occurrences of `static propTypes = {`
* ~104 occurrences of `<COMPONENT>.propTypes = {`

Do we have any coding standards/guide about where `propTypes` should be defined? (so that we can try to get some consistency across the codebase)

Reply:
I think more of the static propTypes is left because we didn't work on them yet.
Check [here](https://github.com/Sage/carbon/wiki/Components)

Reply:
Many thanks for that wiki link. I'll change `static propTypes` to `<COMPONENT>.propTypes`.

Reply:
Updated in bfe2072 :thumbsup: